tests/kafkatest/services/kafka/kafka.py

```diff
@@ -92,6 +92,7 @@ class KafkaService(KafkaPathResolverMixin, JmxMixin, Service):
 
     def __init__(self, context, num_nodes, zk, security_protocol=SecurityConfig.PLAINTEXT, interbroker_security_protocol=SecurityConfig.PLAINTEXT,
                  client_sasl_mechanism=SecurityConfig.SASL_MECHANISM_GSSAPI, interbroker_sasl_mechanism=SecurityConfig.SASL_MECHANISM_GSSAPI,
+                 client_listener_overrides={}, interbroker_listener_overrides={},
```

Comment:
We should add doc which says that these are listener overrides without the listener prefix since we are adding listener prefix to these,